content/references/rippled-api/public-rippled-methods/nft-methods/nft_buy_offers.md

```diff
@@ -0,0 +1,111 @@
+---
+html: nft_buy_offers.html
+parent: nft-methods.html
+blurb: Get a list of all buy offers for a NFToken.
+labels:
+  - Non-fungible Tokens, NFTs, NFTokens
+---
+# account_nfts
+[[Source]](https://github.com/ripple/rippled/blob/xls20/src/ripple/rpc/handlers/NFTOffers.cpp "Source")
+{% include '_snippets/nfts-disclaimer.md' %}
+
+The `nft_buy_offers` method returns a list of buy offers for a `NFToken` object.
+
+## Request Format
+An example of the request format:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+   "id": 1,
+   "command": "nft_buy_offers",
+   "params": [{
+   		"tokenid": "00090000D0B007439B080E9B05BF62403911301A7B1F0CFAA048C0A200000007"
```

Comment:
This line mixes tab and space characters. There are several other examples in the files as well. Please don't use tab characters.